recipes/pari/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "2.9.1" %}
+{% set name = "pari" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://pari.math.u-bordeaux.fr/pub/pari/unix/{{ name }}-{{ version }}.tar.gz
+  sha256: dc510f96686463c1ade7c2a6e16fa9466fd4af6a65fddf9822ba07d7d2e70767
+  patches:
+    - patches/get_dlcflags.patch
+    - patches/prot_none.patch
+    - patches/stackwarn.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - libtool
+    - gmp
```

Comment:
Should we pin this @isuruf? If so can you suggest a pinning in https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py